### ISSUE TITLE
IAM Permissions for FrontEnd to Refresh GHA ASG Instances

### ISSUE THREAD

**Issue by omgitsbillryan** [department-of-veterans-affairs/va.gov-team#34216]
## The Problem
Frontend folks do not have proper IAM access to perform ASG instance refresh - specifically for github runner instances. FE would like to be able to perform this action w/o having to go through OPs for support.

## Extra Context
- [slack thread](https://dsva.slack.com/archives/CJYRZK2HH/p1639591611483100)

## Work to be Done
- [ ] Identify necessary IAM permissions for instance refresh
- [ ] Identify which teams FE IAM teams/groups need these permissions
- [ ] Write IAM in Terraform
- [ ] Apply IAM permissions

**Comment by jhouse-solvd** [comment]
@timwright12 - Is this still needed?

**Comment by timwright12** [comment]
@jhouse-solvd first time I’ve seen this one, but @oseasmoran73 might know

**Comment by oseasmoran73** [comment]
I was trying to see if Bill started or completed the PR and did not find. This request was more to alleviate Ops at the time for refreshing some of FE runners for the hiccups we got from time to time. 

In FE documentation I wrote regarding GHA, I still recommend the route for Ops to refresh if teammates were unfamiliar on how to restart (I wrote some documentation on it), but given that @mydesignrocks is now on their team, this request might not be fully necessary anymore.

Closing ticket unless there is a strong urgency of adding the entire FE team to be able to refresh ASG

CC @timwright12 @mydesignrocks @jhouse-solvd @rmessina1010
